src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -471,6 +477,7 @@ Containers::Optional<SceneData> UfbxImporter::doScene(UnsignedInt) {
     Containers::ArrayView<Vector3d> translations;
     Containers::ArrayView<Quaterniond> rotations;
     Containers::ArrayView<Vector3d> scalings;
+    Containers::ArrayView<UnsignedByte> visibilities; /* @todo should be bool */
```

Comment:
Just FYI, I don't have `SceneFieldType::Bool` because I plan to solve that with some yet-to-be-written `StridedBitArrayView` (i.e., one bit per boolean, not 8).

Didn't get to that yet though, so this workaround is fine in the meantime ;)

Reply:
Yea I did some digging and reached the same conclusion :D 

Is the inclusion of that custom field otherwise reasonable? I thought it would be nice to have before someone starts wondering why they have some dummy objects when importing. One option would be to filter invisible nodes out in the importer but that would complicate everything quite a bit, as the mappings are not 1:1 with ufbx anymore then.

Reply:
Yes, it's perfect like this.

When I have the `StridedBitArrayView`, it'll allow me to add also a builtin `SceneField::Visibility`, and then some `SceneTools` thingy to filter away invisible/dummy/useless objects. Doing that in the importer implementation itself seems painful, and is not something that should be a responsibility of the plugin.

Reply:
Sounds great! The field should be general enough to deserve promotion.